src/App.js

```diff
@@ -1,23 +1,75 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import contacts from "../src/contacts.json";
+import React, { useState } from "react";
 
 function App() {
+  const [contactsList, setContactsList] = useState(contacts.slice(0, 5));
+
+  const addRandom = () => {
+    const random = contacts[Math.floor(Math.random() * contacts.length)];
+    setContactsList([random, ...contactsList]);
```

Comment:
A good idea here is to apply Math.random only on the remaining contacts on the list.
The way you did you might get repeated contacts.